### ISSUE TITLE
Is RoundedTB dead?

### ISSUE THREAD

**Issue by MetalmanSB** [torchgm/RoundedTB#330]
I think it is time we ask "Is RoundedTB DEAD??"

Seeing that RoundedTB has had ZERO updates since November 2021, and that we never got an update for the Windows 22H2 update last year even though there are SEVERAL issues, and now EVEN MORE severe issues with Windows 23H2 releasing, it begs the question, will this ever be updated again??

I understand the devs are working on other projects and have other personal things to deal with but I would figure with how popular RoundedTB is in the custom Windows world that the devs would put some work into making RoundedTB work as flawlessly as possible. And seeing we have not had any updates in almost 2 years is extremely disappointing.

I have been a fan of RoundedTB since Windows 11 launched. I went from using the official release of RoundedTB to taking a break from it when the 22H2 update borked RoundedTB and when Gniang's canary version came out I used that right away and have been using it since then. Gniang's canary build is the only version that seems to be just a bit workable for a triple monitor setup like mine (with some minor issues). But now that Windows 23H2 is releasing, not even the canary build is useable as others have pointed out that the taskbar is not sizing properly and it is cutting off the Start Menu button and some of the pinned apps I have. Now that more severe issues are arising, it makes me wonder if this is the end of the road for RoundedTB. Like I have tweeted at Torch a few times over the past year or so with no response each time as Torch is never online, like EVER.

If this is really the end, I know me and A LOT of others will be very upset that this project was just abandoned without notice. Torch put out a statement on Twitter (X) back in JANUARY of this year stating he will try work on RoundedTB if he can but clearly he has not. It is now almost October and still no updates. I hope Torch sees this at some point and takes note of our frustration. Even if he does not want to work on this project anymore, at least pass it to someone else who would take care of this app.

I know I speak for everyone here when I say we all want an update and want this app to keep thriving. It will be disheartening if it goes on completely abandoned. If it is, our only hope would be that maybe, just maybe, the Windows dev team creates a "dynamic" looking taskbar natively in the near future, maybe in time for Windows 12 release. But we all know how the Windows dev team is like, so chances of that happening is slim to none. I know at least for me, the standard full taskbar is at least 10 years outdated and a native "dynamic" style taskbar is the way of modernizing an old UI.

I know this post is not a technical "issue" and more of a general inquiry but us users deserve to know what is going on here. If ANY devs want to chime in on this convo please do and help answer the question on if RoundedTB is fully flatlined or not. To show how concerned I am about this project, I literally just now created my first GitHub account just to post this message. I hope we all can get some clear answers as to what is going on.

Thanks for listening.

**Comment by bvsr365** [comment]
yes new issue aftter update, thanks @MetalmanSB for expressing your frustration.
